tasks/add_new_openwrap_partner.py

```diff
@@ -1185,7 +1240,10 @@ def main():
 
   if  (setup_type == constant.ADPOD and deal_lineitem_enabled == True) and lineitem_type != constant.LI_SPONSORSHIP:
      raise  BadSettingException('DFP_LINEITEM_TYPE should be SPONSORSHIP for creating deal lineitems')
- 
+
+  if (setup_type == constant.ADPOD and deal_lineitem_enabled == True) and (deal_config_type is None or (deal_config_type != constant.DEALTIER and deal_config_type != constant.DEALID)):
+     raise  MissingSettingException('DEAL_CONFIG_TYPE should be DEALTIER or DEALID')
```

Comment:
We can use the constants defined in the error: 
`constant.DEALTIER `and `constant.DEALID`

Reply:
Done